package.json

```diff
@@ -26,7 +26,7 @@
     "bootswatch": "^4.5.0",
     "classnames": "^2.2.6",
     "date-fns": "^2.12.0",
-    "dayjs": "^1.8.35",
+    "dayjs": "^1.10.4",
```

Comment:
@mike-lvov What was the need for bumping this version here (if any). Generally it would be good to keep version updates as separate PRs, or at the very least, call out why it was needed for the changes made in this PR.